### PR TITLE
Update Ferdi/Ferdium names in Migration guide

### PR DESCRIPTION
**Pull request by Rabattkarte**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -20,15 +20,15 @@ Depending on where you have your account, you have to visit separate sites to re
...
 2. Go to `Help > Import/Export Configuration Data` which should open the corresponding url in your default browser.
-3. Click on `export your data to a ".ferdium-data" file`.
-4. Save the `export.fedi-data` file to anywhere you'd like for safe-keeping/backup.
+3. Click on `export your data to a ".ferdi-data" file`.
+4. Save the `export.ferdi-data` file to anywhere you'd like for safe-keeping/backup.
 
 ## Importing
 
-1. Have Ferdi running on your system.
+1. Have Ferdium running on your system.
```
Comment (vraravam) [inline MIGRATION.md:28]: Importing is only supported into Ferdium - not Ferdi. Please revert the changes in the `import` section.

Comment (vraravam) [reply MIGRATION.md:28]: maybe it can apply on to this line - if the thought process is for "export from ferdi, import into ferdium".

**Review by vraravam — CHANGES_REQUESTED** [review]

File: MIGRATION.md (hunk 1 above)
Comment (Rabattkarte) [reply MIGRATION.md:28]: This doc is about migration, titled MIGRATION.md. My thought process, therefore, is "Migration from Ferdi to Ferdium".

The introduction also states this:

> This guide is here to show you how to export your current Ferdi setup from the online server or from an accountless session for safe-keeping, backup or transferring to a new instance using only a browser.

For importing Ferdium must be running since I am importing the Ferdi data into Ferdium. Or am I mistaken?

---

> Importing is only supported into Ferdium - not Ferdi. Please revert the changes in the import section.

Does not make any sense to me; what exactly do you mean?

Comment (vraravam) [reply MIGRATION.md:28]: sorry - i misread your change (in reverse order - ie Ferdium to Ferdi)

**Comment by Rabattkarte** [comment]
On second thought, maybe we need two documentation?

1. Migration from server-backed to a local account.
2. Migration from Ferdi to Ferdium.

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@all-contributors please add @Rabattkarte for doc
